src/main/webapp/app/guided-tour/guided-tour.service.ts

```diff
@@ -167,13 +161,32 @@ export class GuidedTourService {
             if (this.currentTour.skipCallback) {
                 this.currentTour.skipCallback(this.currentTourStepIndex);
             }
-            this.updateGuidedTourSettings(this.currentTour.settingsKey, this.currentTourStepDisplay, GuidedTourState.STARTED).subscribe(guidedTourSettings => {
+            this.subscribeToAndUpdateGuidedTourSettings(false);
```

Comment:
Not urgent for this hotfix:
It would be better if callers of the method could use the GuidedTourState enum as a parameter, otherwise this is hard to read (what does the 'false' mean here?)